packages/engine/Source/Core/srgbToLinear.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Converts the value from sRGB color space to linear color space.
+ *
+ * @function
+ *
+ * @param {number} value The color value in sRGB color space.
+ * @returns {number} Returns the color value in linear color space.
+ *
+ * @example
+ * const srgbColor = [0.5, 0.5, 0.5];
+ * const linearColor = srgbColor.map(function (c) {
+ *     return Cesium.srgbToLinear(c);
+ * });
+ */
+function srgbToLinear(value) {
+  if (value <= 0.04045) {
+    // eslint-disable-next-line no-loss-of-precision
+    return value * 0.07739938080495356037151702786378;
```

Comment:
We got the implementation for srgbToLinear conversion from existing Cesium engine code. But that function is private and not available for other classes. We just created new function with the same implementation in the Core package. We think a lower precision won’t make any visual difference but tried to follow the Cesium style. Since we just reused this implementation, we are ready to reduce the precision or change the implementation based on Cesium’s suggestions.

Reply:
Got it. The existing implementation is in gltf-pipeline, so I wouldn't recommend refactoring out the code into a shared function.

Ok to leave this as-is.